gamestonk_terminal/portfolio/degiro_api.py

```diff
@@ -0,0 +1,484 @@
+# IMPORTATION STANDARD
```

Comment:
Ideally you would have:
* degiro_controller: What controls the user interaction with degiro
* degiro_view: What takes care of the models and views. Basically what you had before.

So, I think you should have kept the same file you had before as the _view, and add a new one to be the controller. If that makes sense

Reply:
Will keep that in mind for the next version.

Reply:
Will you work on this on the next PR?

This one is kind of important, because we're trying to get the code in this MVC pattern. And currently we have all on same file, which can be a bit confusing if other developers want to develop something on top or fix something

Reply:
Yeah will try to.